src/cast_core/src/RollingCollections.cpp

```diff
@@ -62,8 +62,13 @@ static std::string getRollingCollectionName() {
     // The id is tracked globally and increments for every collection created.
     static std::atomic_long id = 0;
     std::stringstream ss;
-    ss << "r"
-       << "_" << id << "_" << getMillisecondsSinceEpoch();
+    /*
+     * Create a collection name something like: 2019-08-30-11:08:19.123_r134.
+     * Such a name should be unique, and the list of such names in lexigraphic
+     * order will be conveniently ordered by time.
+     */
+    ss << timestamp << std::setfill('0') << std::setw(3) << ms.count()
```

Comment:
I think this change breaks the functionality introduced in the Oplog Tailer?

Reply:
You're right - I merged changes incorrectly.  I will fix.